lib/screens/form/form.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_exercise/screens/form/sections/birth_date.dart';
+import 'package:flutter_form_exercise/screens/form/sections/gender.dart';
+import 'package:flutter_form_exercise/screens/form/sections/opinion.dart';
+import 'package:flutter_form_exercise/screens/form/sections/origin_country.dart';
+import 'package:flutter_form_exercise/screens/form/sections/submit.dart';
+import 'package:flutter_form_exercise/screens/form/sections/terms_and_conditions.dart';
+import 'common_values.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_form_exercise/screens/form/sections/name.dart';
+
+class FormPage extends StatefulWidget {
```

Comment:
Under the screens directory, I thought that every file is a page itself. So why not remove the obvious "page" suffix from them. But I can notice that, this mismatch in the file name and the top level class name is creating confusion. This might mean that the file has something more to offer than the top level class itself. Renamed the file.

Reply:
Yes you've got the idea.